src/platform-includes/getting-started-config/apple.mdx

```diff
@@ -99,6 +99,37 @@ SentrySDK.start { options in
 }];
 ```
 
+## Use Sentry with SwiftUI
+
+> This is an experimental feature
+
+Try Sentry library for [SwiftUI](/platforms/apple/performance/instrumentation/swiftui-instrumentation) and track the performance of your views.
+
+You can install it with CocoaPod:
+
+```ruby
+platform :ios, '13.0'
+use_frameworks! # This is important
+
+target 'YourApp' do
+  pod 'SentrySwiftUI'
+end
+```
+
+Or by importing sentry-cocoa repository (https://github.com/getsentry/sentry-cocoa) with SPM.
+
+This is how you use it:
```

Comment:
```suggestion
In order to use it, you'll need to wrap your view with `SentryTracerView`, like this:
```
Maybe I'm not understanding why this part was dropped, but if it still applies, I think it's helpful to leave it in :)